src/components/PF1SignupForm.tsx

```diff
@@ -1,12 +1,22 @@
+import {
```

Comment:
At this point, shouldn't PF1SignupForm and GMSignupForm be the same? The difference for PF would be the PF information, but that's its own form now right?